src/lib/components/List/List.scss

```diff
@@ -0,0 +1,8 @@
+@import 'vanilla-framework/scss/base';
+@import 'vanilla-framework/scss/patterns_lists';
+@include vf-b-typography;
+@include vf-p-lists;
+
+.p-inline-list--middot .p-inline-list__item::after {
+  right: -0.8rem;
```

Comment:
Is this a fix?

Reply:
Just a reminder - this project is an implementation of Vanilla Framework, not an augmentation of it. Vanilal famowrk should remain the single source of truth for all styles.

Reply:
I agree, I think this repo should have no CSS in it.

Reply:
@anthonydillon Got it - I'll remove the css. I'm not sure why the dots on the middot list are showing up out of place though.

Reply:
Not sure, it should be a Vanilla issue. Could you file an issue please?